region/maridia/inner-yellow.json

```diff
@@ -4994,12 +4994,14 @@
                   "name": "Frozen Wall Pirate Plasma Escape",
                   "notable": false,
                   "requires": [
-                    "canUseFrozenEnemies",
+                    "canTrickyUseFrozenEnemies",
                     "canPreciseWalljump"
                   ],
                   "note": [
                     "It's possible to climb up with just ice by freezing the wall space pirates (who can be frozen and killed without Plasma).",
-                    "Clearing the grey door lock would require killing the frozen pirate after using it to climb."
+                    "Clearing the grey door lock would require killing the frozen pirate after using it to climb.",
+                    "To do this most easily, freeze the Pirate as high as possible on the left wall.",
```

Comment:
```suggestion
                    "To do this most easily, freeze the Pirate as high as possible on the left wall.",
                    "Bait the pirate to jump across the room early so that it stays high on the wall, or bring it on camera by jumping below the platform if the wall pirate gets too low.",
```